specificatie/common.yaml

```diff
@@ -0,0 +1,349 @@
+            instance: https://datapunt.voorbeeldgemeente.nl/api/v1/resourcenaam?parameter=waarde
+            code: notAvailable
+    'default':
+      description: Er is een onverwachte fout opgetreden
+      headers:
+        api-version:
+          $ref: "#/components/headers/api_version"
+      content:
+        application/problem+json:
+          schema:
+            $ref: "#/components/schemas/Foutbericht"
+  schemas:
+    Foutbericht:
+      type: object
+      description: Terugmelding bij een fout. JSON representatie in lijn met [RFC7807](https://tools.ietf.org/html/rfc7807).
+      properties:
+        type:
+          type: string
+          format: uri
+          description: Link naar meer informatie over deze fout
+        title:
+          type: string
+          pattern: ^[a-zA-Z0-9À-ž \.\-]{1,80}$
+          description: Beschrijving van de fout
+        status:
+          type: integer
+          minimum: 100
+          maximum: 600
+          description: Http status code
+        detail:
+          type: string
+          pattern: ^[a-zA-Z0-9À-ž \.\-]{1,200}$
+          description: Details over de fout
+        instance:
+          type: string
+          format: uri
+          description: Uri van de aanroep die de fout heeft veroorzaakt
+        code:
+          type: string
+          pattern: ^[a-zA-Z0-9]{1,25}$
+          description: Systeemcode die het type fout aangeeft
+          minLength: 1
+    InvalidParams:
+      type: object
+      description: Details over fouten in opgegeven parameters
+      properties:
+        type:
+          type: string
+          format: uri
+          example: "https://www.vng.nl/realisatie/api/{major-versie}/validaties/integer"
+        name:
+          type: string
+          pattern: ^[a-zA-Z0-9\.,_]{1,30}$
+          description: Naam van de parameter
+          example: "verblijfplaats__huisnummer"
+        code:
+          type: string
+          pattern: ^[a-zA-Z0-9\.,_]{1,25}$
+          description: Systeemcode die het type fout aangeeft
+          minLength: 1
+          example: "integer"
+        reason:
+          type: string
+          pattern: ^[a-zA-Z0-9\.,_]{1,80}$
+          description: Beschrijving van de fout op de parameterwaarde
+          example: "Waarde is geen geldige integer."
+    BadRequestFoutbericht:
+      allOf:
+      - $ref: "#/components/schemas/Foutbericht"
+      - type: "object"
+        properties:
+          invalidParams:
+            description: Foutmelding per fout in een parameter. Alle gevonden fouten worden één keer teruggemeld.
+            type: array
+            items:
+              $ref: "#/components/schemas/InvalidParams"
+    Waardetabel:
+      type: "object"
+      properties:
+        code:
+          type: "string"
+          pattern: ^[a-zA-Z0-9 \.]+$
+          example: "6030"
+        omschrijving:
+          type: "string"
+          pattern: ^[a-zA-Z0-9À-ž \,\(\)\.\-]{1,200}$
```

Comment:
hier moet wel echt een ' (enkel aanhalingsteken) in mogen, want anders is 's-Gravenhage niet toegestaan of 's-Hertogenbosch